assets/airports/rjbb.json

```diff
@@ -0,0 +1,1958 @@
+                "HABAR": ["WENDY", "SHION", "HABAR"],
+                "CHIZU": ["KAWAT", "CHIZU"]
+            },
+            "draw": [
+                ["HELEN", "MAIKO"],
+                ["B6R11", "HELEN"],
+                ["BB430", "HELEN"],
+                ["MAIKO", "WENDY", "SHION*"],
+                ["SHION", "SOUJA*"],
+                ["SHION", "WASYU*"],
+                ["SHION", "HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "IWAYA1": {
+            "icao": "IWAYA1",
+            "name": "Iwaya One",
+            "rwy": {
+                "24L": ["BB430", ["HELEN", "A80+"], "MAIKO"],
+                "24R": ["BB430", ["HELEN", "A80+"], "MAIKO"]
+            },
+            "exitPoints": {
+                "SHION": ["WENDY", "SHION"],
+                "SOUJA": ["WENDY", "SHION", "SOUJA"],
+                "WASYU": ["WENDY", "SHION", "WASYU"],
+                "HABAR": ["WENDY", "SHION", "HABAR"],
+                "CHIZU": ["KAWAT", "CHIZU"]
+            },
+            "draw": [
+                ["HELEN"],
+                ["SHION*"],
+                ["SOUJA*"],
+                ["WASYU*"],
+                ["HABAR*"],
+                ["CHIZU*"]
+            ]
+        },
+        "DAISY1": {
+            "icao": "DAISY1",
+            "name": "Daisy One",
+            "rwy": {
+                "06L": ["DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "06R": ["B6R10", ["B6R11", "A25+"], "B6R12", "DAISY", ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24L": ["B4L10", ["BB430", "S210"], ["DAISY", "A60"], ["JULIA", "A80+"], "LINDA", "GLORY"],
+                "24R": [["DAISY", "A60+"], ["JULIA", "A80+"], "LINDA", "GLORY"]
+            },
+            "exitPoints": {
+                "HIBIS": ["HIBIS"]
+            },
+            "draw": [
+                ["DAISY", "JULIA", "LINDA", "GLORY"],
+                ["B6R11", "B6R12", "DAISY"],
+                ["BB430", "DAISY"],
+                ["GLORY", "HIBIS*"]
+            ]
+        }
+    },
+    "stars": {
+        "AKASI": {
+            "icao": "AKASI",
+            "name": "Akasi",
+            "entryPoints": {
+                "TSC": ["TSC"]
+            },
+            "body": ["AKASI", "MAIKO", ["MAYAH", "A40"]],
+            "draw": [
+                ["TSC*", "AKASI", "MAIKO", "MAYAH"]
+            ]
+        },
+        "GATES": {
+            "icao": "GATES",
+            "name": "Gates",
+            "entryPoints": {
+                "TSC": ["TSC"]
+            },
+            "body": [
+                ["GATES", "A40"]
+            ],
+            "draw": [
+                ["TSC*", "GATES"]
+            ]
+        },
+        "DANDE": {
+            "icao": "DANDE",
+            "name": "Dande",
+            "suffix": {"06L":"B", "06R":"A", "24L":"C", "24R":"C"},
+            "entryPoints": {
+                "DANDE": ["DANDE"]
+            },
+            "body": [
+                ["JOLLY", "A80-|A60+"]
```

Comment:
the game currently does not support an altitude range, only an altitude restriction. So only one of these will be honored. I recommend (since its a STAR) just including the `A80-` piece, as that will "force" the aircraft to descend to 8000 as it approaches `JOLLY`.